src/components/Body/index.jsx

```diff
@@ -1,61 +1,72 @@
+    speed: 500,
+    slidesToShow: 3,
+    slidesToScroll: 3,
+  };
   return (
-    <Grid container spacing={3} marginTop={5} marginBottom={13}>
-      {/* Main Content Sections */}
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-        <Typography variant="h1" gutterBottom textAlign={"center"}>
-          New Reviews
-        </Typography>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-        <Grid container spacing={2} xs={12}>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-          <Grid item xs={3}>
-            <Card>
-              {/* Card content here */}
-            </Card>
-          </Grid>
-        </Grid>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={8}>
-      </Grid>
-      <Grid item xs={4} md={4} lg={2}>
-      </Grid>
-    </Grid>
+    // <Grid container spacing={3} marginTop={5} marginBottom={13}>
+    //   {/* Main Content Sections */}
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}>
+    //     <Typography variant="h1" gutterBottom textAlign={"center"}>
+    //       New Posts
+    //     </Typography>
+    //   </Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}>
+    //     <Grid container spacing={2} xs={12}>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //       <Grid item xs={3}>
+    //         <Card>{/* Card content here */}</Card>
+    //       </Grid>
+    //     </Grid>
+    //   </Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={2}></Grid>
+    //   <Grid item xs={4} md={4} lg={8}></Grid>
+    //   {/* <Grid item xs={4} md={4} lg={2}></Grid> */}
+    // </Grid>
+    <>
+      <h1 className="body-post">New Posts</h1>
+      <div className="slider-container">
+        <Slider {...settings}>
+          <div>
```

Comment:
It would be better to style the display of the Card too. on desktop, it's flex-row. but on mobile, it flex-column so all Cards will be aligned vertically.

![flex](https://github.com/project-zusammen/hidden-gem-frontend/assets/67864789/3d397bad-1009-462b-a8ad-d3952af8890b)

Reply:
![Uploading fix.png…]()
I have fix

Reply:
when you're uploading an image on comments, PR description, etc in github, make sure to let the upload finish first before submitting the text.

What I see on your previous comment is:
![Screenshot 2024-03-06 at 14 38 51](https://github.com/project-zusammen/hidden-gem-frontend/assets/46950501/290540ed-94f1-44fa-b7d0-f150204b50d0)

Reply:
The picture is uploaded when it looks like so:
![Screenshot 2024-03-06 at 14 39 11](https://github.com/project-zusammen/hidden-gem-frontend/assets/46950501/38f8d962-1f06-4ef0-a1da-2f737b8f00d6)

Reply:
iya mbak